pinot-query-runtime/src/test/resources/queries/WithStatements.json

```diff
@@ -0,0 +1,81 @@
+{
+  "with_statement_tests": {
+    "psql": "7.8.1",
+    "tables": {
+      "tbl1": {
+        "schema": [
+          {"name": "strCol", "type": "STRING"},
```

Comment:
Can we add select .. from (With as x ( select from tbl)) 

With a as (select .. from tbl) , b as (select .. from a), c as (select ... from  join a,b ) select .. from c.

Reply:
could you be more specific?

- `select .. from (With as x ( select from tbl))` is not a valid syntax.
- `with (...) AS a, with (...) AS b SELECT (...) FROM ` is already added. did you meant you would like to nest the previously with tbl in another WITH?

Reply:
this is added